lib/gpu/context.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of gpu;
+
+/// A handle to a graphics context. Used to create and manage GPU resources.
+///
+/// To obtain the default graphics context, use [getContext].
+class Context extends NativeFieldWrapperClass1 {
```

Comment:
I think you should retain GpuContext for this class. Dart itself does not have namespaces, and namespaced imports `import "package:flutter_gpu/flutter_gpu" as gpu` are uncommon.

Context will commonly collide with other Context types, forcing developers to add import namespaces to disambiguate which will be annoying.

Reply:
Good point. Done.